src/Controls/src/Core/HybridWebView/HybridWebView.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+namespace Microsoft.Maui.Controls
+{
+	/// <summary>
+	/// A <see cref="View"/> that presents local HTML content in a web view and allows JavaScript and C# code to interop using messages.
+	/// </summary>
+	public class HybridWebView : View, IHybridWebView
+	{
+		/// <summary>Bindable property for <see cref="DefaultFile"/>.</summary>
+		public static readonly BindableProperty DefaultFileProperty =
+			BindableProperty.Create(nameof(DefaultFile), typeof(string), typeof(HybridWebView), defaultValue: "index.html");
+		/// <summary>Bindable property for <see cref="HybridRoot"/>.</summary>
+		public static readonly BindableProperty HybridRootProperty =
+			BindableProperty.Create(nameof(HybridRoot), typeof(string), typeof(HybridWebView), defaultValue: "HybridRoot");
```

Comment:
I don't want to accidentally have people use the Resources\Raw folder as their whole hybrid root. I'd wager that in many cases that would cause a lot of problems for apps that have other raw resources and conflciting with other aspects of their app.

I sympathize with the concern of not wanting to "pick names" for such things, but it's already something we do all over the place: Blazor apps use `wwwroot`, and even MAUI has naming conventions for lots of things (the resources themselves, Platforms folders, etc.).

Given that I think that using Resources\Raw directly is a bad practice, and wanting most users to have a friendly default, I think `HybridRoot` is a fine value. And if someone wants to head in a different direction, they can do so (but I would imagine very few would).

Reply:
if we are using wwwroot all lowercase, maybe we can use hybridroot?

Reply:
In the mtg we talked about changing it to be `wwwroot` (but in the same `Resources\Raw` location).